Tests/ArgumentParserGenerateDoccReferenceTests/Snapshots/testColorDoccReference().md

```diff
@@ -32,3 +32,8 @@ color help [<subcommands>...]
 ```
 
 **subcommands:**
+
+
+
+
+
```

Comment:
Likewise in these snapshots…

Reply:
The tests break if the newlines are removed.

The newlines are generated by the code; since I removed whitespace trimming, they must be included in the expected output.